_posts/code/PerfContest/4/canny-run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o xtrace
+
+wget --no-clobber https://johnysswlab.com/downloads/221575.tar.gz
+tar -xf 221575.tar.gz
+
+CC=clang
+CFLAGS="-O3 -lm -g -ffast-math"
```

Comment:
I would suggest removing `-g` from the compilation options and add `-march=core-avx2` as a baseline option. Participants could add `-g` when they will be profiling, but by default, we will evaluate the perf of the binaries without debug info.
```suggestion
CFLAGS="-O3 -ffast-math -march=core-avx2 -lm"
```

Reply:
It's fine for march=core-avx2, although I need to test it with clang, I know I had an issue with that line option